src/scope-manager.js

```diff
@@ -315,7 +315,10 @@ var scopeManager = function(state, predefined, exported, declared) {
           isImmutable = labelType === "const" ||
             (labelType === null && _scopeStack[0]["(predefined)"][usedLabelName] === false);
           if (isUnstackingFunctionOuter && !isImmutable) {
-            state.funct["(isCapturing)"] = true;
+            if (!state.funct["(isCapturing)"]) {
+              state.funct["(isCapturing)"] = [];
```

Comment:
The property name "isCapturing" strongly suggests a boolean value. Since we're now implementing it as an array, a new name is in order. This is tricky to describe accurately and concisely, so I'm open to suggestion. My initial thought is "capturedMutables", but I'm not sold on that.

Reply:
"outerMutables"?

Reply:
Even better!